src/metabase/driver/sql.clj

```diff
@@ -63,6 +63,17 @@
       (seq referenced-card-ids)
       (update :metabase.models.query.permissions/referenced-card-ids set/union referenced-card-ids))))
 
+(defmulti json-field-length
+  "Return a HoneySQL expression that calculates the length of a JSON field for a given driver.
```

Comment:
I would just clarify that this is the textual length, not the "collection length" (i.e. the number of top-level keys).

I assumed it was the latter before I read the code 😄

Reply:
yep, used "calculates the number of characters in a JSON field" 9b04fb0ca9d8d98a636088162cdbc4eb3f319e78